src/components/VSelect/VSelect.vue

```diff
@@ -329,7 +329,21 @@
           click: () => {
             if (!this.isActive) this.isActive = true
           },
-          keydown: this.onKeyDown // Located in mixins/autocomplete.js
+          keydown: (e) => {
+            if (!this.isActive &&
+                !this.multiple &&
```

Comment:
I'm not clear on what that means though unless I misunderstood? If a multiple select has A, B, C selected, which one does it increment when the up/down key is pressed?